Nomad_VanilaJS/Nomad_VanilaJS_JeongBoWoon/js/quotes.js

```diff
@@ -0,0 +1,18 @@
+var quotes = [
+    {quote:"밥먹으면 배부르다", author:"아무개"},
+    {quote:"가만히만 있어도 반은 가더라", author:"아무개"},
+    {quote:"사랑은 가까이에 있다", author:"정예원"},
+    {quote:"티끌모아 티끌.", author:"김란주"},
+    {quote:"지금 과제할 시간도 없는데 어떻게 놀아!!!", author:"영경배"},
+    {quote:"졸리면 자야한다", author:"아무개"},
+    {quote:"웃으면 복이오진 않지만 기분이 좋아진다", author:"아무개"},
+    {quote:"하기 싫으면 안한다", author:"정예원"},
+];
+
+const quoteContent = document.querySelector("#quoteForm span:first-child");
+const author = quoteForm.querySelector("#quoteForm span:last-child")
+
+const todayQuote = quotes[Math.floor(Math.random()*8)];
+
+quoteContent.innerText=todayQuote.quote;
+author.innerText=todayQuote.author;
```

Comment:
강의에서는 가상 선택자를 사용하는 방법을 알려주기 위해서 이런 방법을 사용한 것 같아요!
하지만 실제로 이런 코드의 경우 다음과 같이 작성할 수도 있어요.

```suggestion
const quote = document.querySelector("#quote");
const todayQuote = quotes[Math.floor(Math.random()*8)];

quote.innerHTML = `<span>%{todayQuote.quote} - %{todayQuote.author} </span>`;
```